app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -346,21 +346,15 @@ def template_from_vmdb(deployment)
       end
 
       def download_template(uri)
-        require 'open-uri'
+        options = {
+          :method      => 'get',
+          :url         => uri,
+          :proxy       => @config.proxy,
```

Comment:
@blomquisg Yes. It's normally a single URL. It was open-uri that made us split them out.